crates/subspace-networking/src/constructor/transport.rs

```diff
@@ -24,15 +23,11 @@ pub(super) fn build_transport(
     timeout: Duration,
     yamux_config: YamuxConfig,
 ) -> io::Result<Boxed<(PeerId, StreamMuxerBox)>> {
-    let wrapped_tcp = {
-        let tcp_config = GenTcpConfig::default().nodelay(true);
```

Comment:
it seems like this is still required, because the merged PR got the setting the wrong way around, and used `Some(false)`:
https://github.com/libp2p/rust-libp2p/pull/5469/files#r1758528479

Reply:
lol